src/transformers/pipelines/text2text_generation.py

```diff
@@ -50,6 +52,31 @@ def check_inputs(self, input_length: int, min_length: int, max_length: int):
         """
         return True
 
+    def _parse_and_tokenize(self, *args, truncation):
+        prefix = self.model.config.prefix if self.model.config.prefix is not None else ""
+        if isinstance(args[0], list):
+            assert (
+                self.tokenizer.pad_token_id is not None
+            ), "Please make sure that the tokenizer has a pad_token_id when using a batch input"
+            args = ([prefix + arg for arg in args[0]],)
+            padding = True
+
+        elif isinstance(args[0], str):
+            args = (prefix + args[0],)
+            padding = False
+        else:
+            raise ValueError(
+                " `args[0]`: {} have the wrong format. The should be either of type `str` or type `list`".format(
+                    args[0]
+                )
+            )
+        inputs = super()._parse_and_tokenize(*args, padding=padding, truncation=truncation)
+        # This is produced by tokenizers but is an invalid
+        # generate kwargs
+        if "token_type_ids" in inputs:
+            del inputs["token_type_ids"]
+        return inputs
```

Comment:
can we instead pass `retuen_tokens_types_ids=False` as kwarg to `_parse_and_tokenize` ?

Reply:
Is it on by default now ?

Reply:
It is on for BERT like models, but It shouldn't be for seq2seq models. What model was failing because of this?

Reply:
The translation tests I added in this PR were faling